weapon-config.inc

```diff
@@ -3526,18 +3532,20 @@ public OnPlayerWeaponShot(playerid, weaponid, hittype, hitid, Float:fX, Float:fY
 				new Float:health;
 
 				GetVehicleHealth(hitid, health);
+				if(health >= 249.0) { //vehicles start on fire at 250 or under so theres no need to check once the vehicle is at or below 250
```

Comment:
I'm pretty sure a vehicle can have `250.0` and not burn. Anything below that means it's on fire.